src/App.tsx

```diff
@@ -1,6 +1,8 @@
 /**
  * Entry function from main.tsx which will be the main entry point for the differing Components
  */
-const App = () => {};
+const App = () => {
```

Comment:
remove before merging to main

Reply:
sounds good, just a place holder to make sure its working